State/ApplicationState.cpp

```diff
@@ -80,6 +88,8 @@ Concurrency::task<void> moonlight_xbox_dx::ApplicationState::UpdateFile()
 		stateJson["alternateCombination"] = that->AlternateCombination;
 		for (auto host : that->SavedHosts) {
 			nlohmann::json hostJson;
+			hostJson["ipAddress"] = Utils::PlatformStringToStdString(host->IPAddress);
```

Comment:
What happens when a user with an older version of the app opens it? 
ipAddress/port might not be populated when upgrading